lib/timers.js

```diff
@@ -182,6 +191,8 @@ exports.active = function(item) {
       item._monotonicStartTime = Timer.now();
       L.append(list, item);
     }
+
+    list = lists[msecs];
```

Comment:
@saquibkhan It is not needed, I forgot to remove it from the previous patch, good catch :+1: and thank you for taking the time to review this change :)